cni/deployments/kubernetes/Dockerfile.install-cni

```diff
@@ -1,15 +1,17 @@
-FROM ubuntu:xenial
+# BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
+ARG BASE_DISTRIBUTION=default
```

Comment:
Do we plan to add distroless option as well? If so we need something like https://github.com/istio/istio/blob/7017f7575c0d22daffddbca1a55fe62f3984dcf4/pilot/docker/Dockerfile.pilot#L15

Reasonable to defer to another PR though